marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 5842035442347632,
+            "layout": {
+                "height": 26,
+                "width": 15,
+                "x": 112,
+                "y": 25
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "avg:marklogic.forests.query_read_rate{*}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Read query rate",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 779967061471034,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 73,
+                "y": 38
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "avg:marklogic.forests.compressed_tree_cache_hit_rate{*}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Hit rate of the compressed tree cache",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 2378507056004724,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 73,
+                "y": 22
+            }
+        },
+        {
+            "definition": {
```

Comment:
I don't think we need this label—I think readers will probably be looking for unusual traffic spikes already if they're watching this graph.